### PR TITLE
FIX: cv builder

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by scopatz** [comment]
looks like there are some test failures

**Comment by CJ-Wright** [comment]
Hmm, what is the best way to deal with URLs? One of the changes were to launder URLs through `latex_safe` will that harm them?

**Comment by scopatz** [comment]
URLs should probably be wrapped In \href or \url

**Comment by CJ-Wright** [comment]
In the database? Or should we try to catch it in the builder/jinja layer?

**Comment by scopatz** [comment]
Probably in the builder

**Comment by CJ-Wright** [comment]
I guess I should build a URL regex and try to catch things that look URL like and wrap them. Maybe attach it to latex_safe.

**Comment by scopatz** [comment]
That would be awesome!

**Comment by CJ-Wright** [comment]
@scopatz ready for review!

File: tests/outputs/cv/scopatz.tex [hunk 1]
```diff
@@ -346,7 +346,7 @@
       
       
         \resitem{
-            \href{https://docs.google.com/document/d/1uMAx_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
+            \href{https://docs.google.com/document/d/1uMAx\_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
```
Comment (CJ-Wright) [inline tests/outputs/cv/scopatz.tex]: @scopatz what is the god behavior here?

File: regolith/templates/resume.tex [hunk 2]
```diff
@@ -208,13 +208,13 @@
       {% if 'narrative' in grant or 'benefit_of_collaboration' in grant or 'call_for_proposals' in grant %}
         \resitem{
           {%- if 'narrative' in grant %}
-            \href{ {{-latex_safe(grant['narrative'])-}} }{Narrative}.
+            \href{ {{-latex_safe(grant['narrative'], url_check=False)-}} }{Narrative}.
           {%- endif %}
           {%- if 'benefit_of_collaboration' in grant %}
-            \href{ {{-latex_safe(grant['benefit_of_collaboration'])-}} }{Benefit of Collaboration}.
+            \href{ {{-latex_safe(grant['benefit_of_collaboration'], url_check=False)-}} }{Benefit of Collaboration}.
```
Comment (CJ-Wright) [inline regolith/templates/resume.tex]: Maybe we should remove latex_safe from things that are URLs outright?

File: tests/outputs/cv/scopatz.tex (hunk 1 above)
Comment (scopatz) [reply tests/outputs/cv/scopatz.tex]: You should not escape the underscore in hrefs: https://tex.stackexchange.com/questions/53399/is-it-necessary-to-escape-underscores-in-href-with-dollar-signs-or

File: regolith/templates/resume.tex (hunk 2 above)
Comment (scopatz) [reply regolith/templates/resume.tex]: yeah, probably

**Comment by CJ-Wright** [comment]
@scopatz I think this is ready for another round.

**Pull request merged**